ui/components/multichain/product-tour-popover/product-tour-popover.scss

```diff
@@ -0,0 +1,32 @@
+.multichain-product-tour-menu {
+  width: 344px;
+  box-shadow: none;
+  left: -7px !important;
+  top: 10px !important; //important is required here since Menu has absolute position added via inline style in base component.
```

Comment:
Thank you for leaving an explanation for the `important!` usage! This will help with maintainability!

Reply:
I wonder how we can accommodate RTL here....

Reply:
@darkwing I tried converting Permissions to Arabic and this is how it looks. I am not sure what you meant by RTL accommodation here 🙈 . Since with this Left value, we are just controlling the box position 🤔 
https://user-images.githubusercontent.com/39872794/233020677-600560eb-83a9-4ef7-8b21-5006f32d7892.mov